src/core/domain/repositories/pedidoRepository.interface.ts

```diff
@@ -0,0 +1,7 @@
+import { Pedido } from "@domain/entities/pedido";
+
+export interface IPedidoRepository {
+    getPedidos(filters?: Partial<Pedido>): Promise<Pedido[]>;
+    getPedidoById(id: string): Promise<Pedido>;
+    updatePedido(id: string, pédido: Partial<Pedido>): Promise<Pedido>;
```

Comment:
Aqui acabou ficando um acento no pedido do parâmetro
```suggestion
    updatePedido(id: string, pedido: Partial<Pedido>): Promise<Pedido>;
```